README.md

```diff
@@ -223,6 +223,15 @@ facts:
   environment: 'staging'
 ```
 
+## Marathon Defaults
```

Comment:
I love this. We can put this at the top of the readme and remove all the mentions of "-m http://marathon:8080" since it's now optional, the default would be to put it the config files.